tests/cursor/bug1713-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+MongoDB\Driver\Cursor PHPC-171: The cursor current method does not return anything
```

Comment:
By convention, the JIRA ticket is used as the test name prefix. Non-ticket tests start with a class/method name. I'd suggest:

> PHPC-1713: MongoDB\Driver\Cursor::current() does not return anything